### DISCUSSION TITLE
Is it possbile to get the silly zone™ back on DTM-16 v5 ?

### DISCUSSION THREAD

**Discussion by Spokelax** [XYZenix/XYZenixThemes#141] [General]
The discord shop gives me high amount of cancer so I really enjoyed the silly zone.
Silly > cringe.

**Comment by XYZenix** [comment]
if you compile DTM-16 with the silly zone scss variable enabled (in `_options.scss`), yes you can have the silly zone back, but since i've had at least one complaint about it and because the gif that was meant to be featured in it no longer works (i don't want to embed an entire gif into the css) i chose to just get rid of it

additionally it probably broke BD's theme rules even if nobody bothered to complain to me about that, so i chose the safer option to just theme the collectibles shop (even though iirc avatar decos and profile effects are still completely hidden in DTM-16 lol)
